hack/tools/logcheck/testdata/data.go

```diff
@@ -23,22 +23,25 @@ import (
-	klog.Warning("test log")         // want `unstructured logging function "Warning" should not be used`
-	klog.Warningf("test log")        // want `unstructured logging function "Warningf" should not be used`
-	klog.WarningDepth(1, "test log") // want `unstructured logging function "WarningDepth" should not be used`
-	klog.Error("test log")           // want `unstructured logging function "Error" should not be used`
-	klog.Errorf("test log")          // want `unstructured logging function "Errorf" should not be used`
-	klog.Errorln("test log")         // want `unstructured logging function "Errorln" should not be used`
-	klog.ErrorDepth(1, "test log")   // want `unstructured logging function "ErrorDepth" should not be used`
-	klog.Fatal("test log")           // want `unstructured logging function "Fatal" should not be used`
-	klog.Fatalf("test log")          // want `unstructured logging function "Fatalf" should not be used`
-	klog.Fatalln("test log")         // want `unstructured logging function "Fatalln" should not be used`
-	klog.FatalDepth(1, "test log")   // want `unstructured logging function "FatalDepth" should not be used`
+	klog.V(1).Infof("test log")         // want `unstructured logging function "Infof" should not be used`
+	klog.Infof("test log")              // want `unstructured logging function "Infof" should not be used`
+	klog.Info("test log")               // want `unstructured logging function "Info" should not be used`
+	klog.Infoln("test log")             // want `unstructured logging function "Infoln" should not be used`
+	klog.InfoDepth(1, "test log")       // want `unstructured logging function "InfoDepth" should not be used`
+	klog.Warning("test log")            // want `unstructured logging function "Warning" should not be used`
+	klog.Warningf("test log")           // want `unstructured logging function "Warningf" should not be used`
+	klog.WarningDepth(1, "test log")    // want `unstructured logging function "WarningDepth" should not be used`
+	klog.Error("test log")              // want `unstructured logging function "Error" should not be used`
+	klog.Errorf("test log")             // want `unstructured logging function "Errorf" should not be used`
+	klog.Errorln("test log")            // want `unstructured logging function "Errorln" should not be used`
+	klog.ErrorDepth(1, "test log")      // want `unstructured logging function "ErrorDepth" should not be used`
+	klog.Fatal("test log")              // want `unstructured logging function "Fatal" should not be used`
+	klog.Fatalf("test log")             // want `unstructured logging function "Fatalf" should not be used`
+	klog.Fatalln("test log")            // want `unstructured logging function "Fatalln" should not be used`
+	klog.FatalDepth(1, "test log")      // want `unstructured logging function "FatalDepth" should not be used`
+	klog.InfoS("test: %s", "testname")  // want `structured logging function "InfoS" should not use format specifier "%s"`
```

Comment:
Move this calls to `printStructuredLog()` function

Reply:
Done.